addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -330,9 +326,6 @@ function factory(dependencies) {
          * @param {MouseEvent} ev
          */
         onMouseEnterTopbarThreadDescription(ev) {
-            if (!this.thread || !this.thread.isChannelDescriptionChangeable) {
```

Comment:
Keep condition `!this.exists()` return to be safe (that was implicit in the previous check `!this.thread`)